### PR TITLE
Wait for form to load when following password reset link

### PR DESCRIPTION
**Pull request by skshetry**
## Description

This adds a wait for the form in the `password reset` page. 

## Motivation and Context

Fixes [drone error](https://drone.owncloud.com/owncloud/core/17030/903)

## How Has This Been Tested?

- locally and :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/lib/LoginPage.php [hunk 1]
```diff
@@ -43,6 +43,7 @@ class LoginPage extends OwncloudPage {
 	protected $submitLoginId = "submit";
 	protected $lostPasswordId = "lost-password";
 	protected $setPasswordErrorMessageId = "error-message";
+	protected $setPasswordFormXpath = '//form[@id="set-password"]';
```
Comment (paurakhsharma) [inline tests/acceptance/features/lib/LoginPage.php]: Can we use Id's here?

**Review by phil-davis — DISMISSED** [review]

**Review by paurakhsharma — DISMISSED** [review]
Looks good :+1:

**Review by phil-davis — DISMISSED** [review]

**Comment by phil-davis** [comment]
@skshetry drone CI failures. The `set-password` id does not seem to exist in all cases.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport `stable10` #35123

**Pull request merged**